### PR TITLE
Fix description error in classification tutorial

### PR DESCRIPTION
**Pull request by amnivek**
The last dense layer is a logits layer, instead of a softmax layer.

### CODE REVIEW

**Comment by amnivek** [comment]
@googlebot I signed it!
